src/components/contact/ContactForm.vue

```diff
@@ -164,7 +164,8 @@ export default class ContactForm extends Vue {
   get formattedCredentialTypes(): Array<{ text: string; value: number }> {
     return this.credentialTypes.map((type) => ({
       text:
-        type.schema_label && type.schema_label.translations[this.$i18n.locale].label
+        type.schema_label?.translations &&
+        type.schema_label?.translations[this.$i18n.locale]?.label
```

Comment:
I think optional chaining (`?.`) also works on arrays so you can just do:

`type.schema_label?.translations?.[this.$i18n.locale]?.label`